.travis.yml

```diff
@@ -1,31 +1,31 @@
 env:
   global:
+    - coverage=false
     - CC_TEST_REPORTER_ID=35fd34991dd17bf1e419bc058c014482759b494aad0f02f19594f39b58950d79
 
 language: php
 
 matrix:
   include:
   - php: 7.0
+    env: coverage=true
   - php: 7.1
   - php: 7.2
+  - php: 7.3
+  - php: 7.4
 
 sudo: false
 
-addons:
-  code_climate:
-    repo_token: 35fd34991dd17bf1e419bc058c014482759b494aad0f02f19594f39b58950d79
-
 install:
-    - composer install
+    - composer update
```

Comment:
Using the `composer update` to avoid `composer` install cached dependencies when executing different PHP versions.